src/easier68k/core/opcodes/MOVE.py

```diff
@@ -0,0 +1,153 @@
+from ...core.enum.ea_mode import EAMode
+from ...core.enum.op_size import BigSSize
+from ...core.enum.ea_mode_bin import EAModeBinary
+from ...simulator.m68k import M68K
+import binascii
+
+# Allowed values: nothing, or some combination of B, W, and L (for byte, word, and long)
+# For example, MOVE would have 'BWL' because it can operate on any size of data, while MOVEA would have 'WL' because
+# it can't operate on byte-sized data
+allowed_sizes = 'BWL'
+
+
+class MOVE:
+    # Include parameters here
+    def __init__(self, src: EAMode, dest: EAMode, size='B'):
+        assert src.mode != EAMode.ARD  # Only invalid src is address register direct
+        self.src = src
+
+        assert dest.mode != EAMode.ARD and dest.mode != EAMode.IMM  # Can't take address register direct or immediates
+        self.dest = dest
+
+        assert size in allowed_sizes
+        self.size = size
+
+    def assemble(self) -> bytearray:
```

Comment:
since this is considered an 'example' opcode, I think we should include more docs so that we have a clear pattern to follow for later ones

Reply:
Will do.